src/Finder/OnlyOnLinuxFinder.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+/*
+ * This file is part of the Fidry CPUCounter Config package.
+ *
+ * (c) Théo FIDRY <theo.fidry@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Fidry\CpuCoreCounter\Finder;
+
+use const PHP_OS_FAMILY;
+
+final class OnlyOnLinuxFinder implements CpuCoreFinder
```

Comment:
Might be worth introducing it in a different PR with the other variants `SkipOnLinux`, `OnlyOnMacOs`, `SkipOnMacOs`. Probably an abstract finder could be extracted from it as well. Even if they are not applied in this PR, I think they are good general purpose finders to have

Reply:
Yup, sounds good.

Reply:
Revisiting this: maybe a simpler way would be `SkipOnOSFamily` and `OnlyOnOSFamily` where you can pass the `string $osFamily`. It could also contain static methods to make it nicer, `SkipOnOSFamily::onWindows(): self`. The existing Windows variant could then be deprecated in favour of those ones.

Reply:
Did that in https://github.com/theofidry/cpu-core-counter/pull/94

Reply:
Ok, resolving this.